README.md

```diff
@@ -5,13 +5,15 @@ Wiki for the Pharo programming language and development environment.
 The goals of the wiki are explained in the [Manifest](MANIFEST.md) and contribution guidelines in the [Contribution guide](CONTRIBUTION.md).
 
 ## Contents
-- [Beginners](#beginners)
-- [General](#general)
-- [Pharo projects](#pharo-projects)
-- [External projects](#external-projects)
-  * [Data exchange](#data-exchange)
-- [Migration guidelines](#migration-guidelines)
-- [External ressources](#external-ressources)
+- [Pharo wiki](#pharo-wiki)
+  - [Contents](#contents)
+  - [Beginners](#beginners)
+  - [General](#general)
+  - [Pharo projects](#pharo-projects)
+  - [External projects](#external-projects)
+    - [Data exchange](#data-exchange)
+    - [Data structures](#data-structures)
```

Comment:
Why this new section? I have the impression we have no section named like this on the page yet.

Reply:
I think it's commented out? 

```
<!--
### Data structures
- [DataFrame](ExternalProjects/DataStructures/DataFrame.md) ![TODO](https://img.shields.io/badge/Progress-TODO-red.svg?style=flat)
-->
```

That is, comments are not really standard in Markdown and the extension parsed that. I will turn this feature off if I can :-)

Reply:
I think I fixed it with another commit